app-admin/rex/rex-9999.ebuild

```diff
@@ -0,0 +1,121 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+if [[ ${PV} == 9999 ]]; then
+	GITHUB_USER=RexOps
+	GITHUB_REPO=Rex
+	EGIT_REPO_URI="https://github.com/${GITHUB_USER}/${GITHUB_REPO}.git"
+	EGIT_BRANCH="development-1.x"
+	VCS_ECLASS="git-r3"
+else
+	# This is intentional to stop perl-module.eclass doing magic things when it
+	# shouldn't. Like making ${S} contain "Rex" when the git clone has "rex"
+	DIST_AUTHOR=JFRIED
+	DIST_NAME=Rex
+fi
+
+inherit perl-module ${VCS_ECLASS}
+
+[[ ${PV} == 9999 ]] && SRC_URI=""
```

Comment:
if you just dropped this, would it be a problem? Does `SRC_URI` need to be defined? Not 100% sure, but I think not (how would virtuals then work?)